protocol-definitions/MC.yaml

```diff
@@ -859,3 +859,37 @@ methods:
          since: 2.4
          doc: Name of the Queue for which the age-related statistics should be reset to 0
     response: {}
+  - id: 34
+    name: reloadConfig
+    since: 2.4
+    doc: Reloads the configuration on the member from its config file, applies (merges) the new config to the current config
+    request:
+      retryable: false
+      partitionIdentifier: -1
+    response:
```

Comment:
do we need to return a result of the update here?
We are going to get status events anyway, may be it makes sense to have this process async